nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/MergeRecord.java

```diff
@@ -171,6 +174,7 @@ public class MergeRecord extends AbstractSessionFactoryProcessor {
     public static final String MERGE_COUNT_ATTRIBUTE = "merge.count";
     public static final String MERGE_BIN_AGE_ATTRIBUTE = "merge.bin.age";
     public static final String MERGE_UUID_ATTRIBUTE = "merge.uuid";
+    public static final String REASON_FOR_MERGING = "merge.reason";
```

Comment:
Recommend adjusting the naming to provide some differentiation from `MergeContent`, which has a different set of defined values.
```suggestion
    public static final String MERGE_COMPLETION_REASON = "merge.completion.reason";
```